packages/nouns-assets/package.json

```diff
@@ -11,10 +11,15 @@
     "encode": "ts-node scripts/encode.ts",
     "clean": "rimraf dist tsconfig.build.tsbuildinfo",
     "build": "tsc -p tsconfig.build.json",
-    "prepare": "yarn build"
+    "prepare": "yarn build",
+    "test": "mocha -r ts-node/register \"test/**/*.test.ts\""
   },
   "devDependencies": {
+    "@ethersproject/bignumber": "^5.5.0",
+    "@ethersproject/solidity": "^5.5.0",
```

Comment:
These will both need to be included in `dependencies` as they're consumed in the build output